src/client/styles/components/Feedback.scss

```diff
@@ -0,0 +1,83 @@
+.feedback {
+  position: fixed;
+  max-width: 400px;
+  bottom: 0;
+  right: 0;
+  z-index: 100;
+  text-align: right;
+}
+
+.feedback-button {
```

Comment:
No super-good reason.  I just have a general aversion to nesting for some reason.  We could do a bit of an audit/refactoring of the css to make things more sane/re-usable.